public/js/index.js

```diff
@@ -12,11 +12,10 @@ const overlay = document.querySelector('.overlay');
 const modal = document.querySelector('.modal');
 const modalLesson = document.querySelector('.modal-lesson');
 const modalLessonClose = document.querySelector('.modal-lesson-close');
+const modalLessonCloseWithESCKey = document.querySelector('.modal-lesson-close');
```

Comment:
Hitting the Esc key is still not closing the modal when I test this out. 

I think we might need to attach the event listener to the document rather than the close button. That way the keydown event will be registered. Let me know if this doesn't make sense 😄